pkg/daemon/ceph/osd/daemon.go

```diff
@@ -154,15 +164,19 @@ func Run(context *clusterd.Context, agent *OsdAgent, done chan struct{}) error {
 	mon := NewMonitor(context, agent)
 	go mon.Run()
 
-	// FIX
-	logger.Infof("sleeping a while to let the osds run...")
 	select {
-	case <-time.After(1000000 * time.Second):
-		logger.Warning("OSD sleep has expired")
 	case <-done:
 		logger.Infof("done channel signaled")
 	}
 
+	// umount devices
+	for m := range mountPoints {
+		logger.Infof("unmount %s", mountPoints[m])
+		if err = sys.UnmountDevice(mountPoints[m], context.Executor); err != nil {
```

Comment:
Mounting a device is only done for a filestore device, right? If we have a bluestore device (the common scenario), we wouldn't need to mount or unmount. And if it is a filestore device, shouldn't this preparation pod unmount it anyway and then the osd pod mount it when it starts?

Reply:
done